sonar-csharp-plugin/src/test/resources/CSharpSensorTest/output-cs/metrics.pb

```diff
@@ -1,4 +1,4 @@
-�
+�
 
 Program.cs (08@HR 0=0;5=1;10=0;20=0;30=0;60=0;90=0Z1=2;2=0;4=1;6=0;8=0;10=0;12=0b1j
-
68r) !"#%&()*+-./12356789:;<>
\ No newline at end of file
+
68r) !"#%&()*+-./12356789:;<>� "%:
```

Comment:
Is it documented how to produce/update this file? (I'm guessing it's used as a known test input).

Reply:
There is a readme in sonar-dotnet-shared-library folder, I'll add a readme with a pointer that that file.